connection.go

```
@@ -43,9 +42,18 @@ func (c *Client) launchConnection() error {
 	quit := c.conn.GetQuit()
 
 	// Launch processes to keep track of connection & data
-	go c.writeWorker(c.err, quit) // Initiates message writing to the Gremlin-server
-	go c.readWorker(c.err, quit)  // Initiates message reading from the Gremlin-server
-	go c.conn.Ping(c.err)         // Manages pinging and connection to the Gremlin-server
+	c.commRoutines.Go(func() error {
```

If you ignore the errors, you can use WG instead.